docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -2,30 +2,53 @@
 
-Assuming you have got the Arktos repo downloaded to your local disk and the current folder is at the root of the repo,
+Assuming you have got the Arktos repo downloaded to your local disk and the current folder (i.e ~/go/src/arktos/) is at the root of the repo, and 
 
-0. Make sure the following directories are empty. If not, clean them up.
+- Setup master node and workwer nodes based on [set up developer environment](setup-dev-env.md) to install needed packages (docker, make, gcc, jq and golang).
+- SSH/SCP should work from worker nodes to master node in order to copy the worker secret files from the master node (in AWS, use private key file of keypair to access to master/worker nodes).
+- Needed ports are opened between master node and worker nodes
+  * Allow access to kube-api port 6443 on master node from work nodes (in AWS, add this rule into inbound rules of security group for master node)
+  * Allow access to kubelet port 10251 and kube-proxy port 10255 on worker nodes from master node(in AWS, add this rule into inbound rules of security group for worker nodes)
+- On master node, the permisson of others for file /var/run/docker.sock should be readable and writable.
+
+```bash
+srw-rw-rw- 1 root docker 0 Aug  3 22:15 /var/run/docker.sock
+```
+
+0. Make sure the following directories are empty. If not, clean them up using sudo permisson
 ```
-/opt/cni/bin/
-/etc/cni/net.d/
+sudo rm -rf /opt/cni/bin/*
+sudo rm -rf /etc/cni/net.d/*
 ```
 
-1. bootstrap the cluster by starting the master node (no CNI plgin at first)
+1. bootstrap the cluster by starting the master node (no CNI plugin at first)
 ```bash
 export ARKTOS_NO_CNI_PREINSTALLED=y
+make clean
 ./hack/arktos-up.sh
 ```
 
-Note: arktos-up.sh should be stuck in "Waiting for node ready at api server" messages. Don't worry, the apiserver is already up at this point, just the master node status is not "Ready" as we have not installed the network plugin yet. 
+Note: arktos-up.sh should be stuck in "Waiting for node ready at api server" messages. Don't worry, the apiserver is already up at this point, just the master node status is not "Ready"  and pod/kube-dns in name space 'kube-system' is not in state of 'Running' as we have not installed the network plugin yet. 
 
-2. Open another terminal to the master node to install CNI plugin
+2. Open another terminal to the master node to install CNI plugin of flannel
 ```bash
 ./cluster/kubectl.sh apply -f https://github.com/coreos/flannel/raw/master/Documentation/kube-flannel.yml
 ```
 
-After that, "arktos-up.sh" should get rid of "Waiting for node ready at api server" messages and be successfully started.
+After that, "arktos-up.sh" should get rid of "Waiting for node ready at api server" messages and be successfully started. You can check the status of all resources including pod/kube-dns and flannel network
+
+```bash
+./cluster/kubectl.sh get all --all-namespaces
```

Comment:
get errors:
```
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```

Reply:
Add the reminder to export "KUBECONFIG" in bash command lines. Thanks!